src/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url('../images/star.svg');
```

Comment:
Ensure that the path '../images/star.svg' is correct and that the image exists at this location. Incorrect paths will result in the background image not being displayed.